stock_picking_partner_invoice_address/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+- Adds partner_invoice_id field to stock picking.
+- Depending on whether the partner_invoiced_id is the same or not, the data to be linked to the shipping company and the template to be used for export will be different.
```

Comment:
> Depending on whether the partner_invoiced_id is the same or not, the data to be linked to the shipping company and the template to be used for export will be different.

This is not the function of the module. This is rather an expected usage which requires manual adjustments on the export layout.